pages/dkp/kommander/1.4/release-notes/legal-notices/index.md

```diff
@@ -3,7 +3,7 @@ layout: layout.pug
 beta: false
 title: Legal Notices
 menuWeight: 105
-excerpt: List of Third-party trademarks mentioned in the Kommander documentation
+excerpt: List of third party trademarks mentioned in the Kommander documentation
```

Comment:
I just did this to be consistent for the landing page. It bugged me ... so ... tada!